### ISSUE TITLE
Paginate get_pages results

### ISSUE THREAD

**Issue by boogheta** [medialab/hyphe#293]

**Comment by paulgirard** [comment]
oh yeah !
And maybe  we should put the page list in a specific tab and not as a default view in the UX in WE page ?

**Comment by boogheta** [comment]
This opens a few questions:
- How to display the total number of known pages in the webentity page without loading them all? Should we drop the display? Or maintain some kind of cache of numbers returned by the traph during indexation or such?
- Should we remove the list of pages from the first view of a webentity page, and only give access to it in a tab/page that would trigger the query?
- There's also no way to order them by indegree without having them all already, so I guess this implies to display them alphabetically instead by default and ordre them only once all were loaded

@jacomyma @Yomguithereal if you think of other relevant elements on the matter!

**Comment by jacomyma** [comment]
The list of pages is a good way to represent the content of a web entity in Hyphe. This is why it is featured currently. But the performance issue it rises is serious, and we could get rid of it. Most of the time, the list itself is not really useful. The number of pages, however, is both an acceptable way to represent the content and an important information to monitor the harvesting process.

Both the number of crawled pages and the number of discovered pages are important. They help diagnose crawling issues or non-issue (eg. when the crawler did not find pages but for good reason) and give a rough idea of the web entity profile.

In a nutshell, although we could put the list of pages in a separate view to have a more responsive UI, we should probably cache the number(s) of pages if we can, so that we can access it efficiently.
